pkg/scheduler/scheduler.go

```diff
@@ -427,7 +427,7 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 		PodSetAssignments: e.assignment.ToAPI(),
 	}
 
-	workload.SetAdmission(newWorkload, admission)
+	workload.SetQuotaReservation(newWorkload, admission)
```

Comment:
We should probably change the name of the `admit` function too, but let's defer that after we have settled on the name.